cypress/integration/requests/requests-sort.spec.js

```diff
@@ -52,7 +51,8 @@ describe('ui-requests: Sort requests', () => {
     Requests.deleteRequestPolicyApi(requestPolicyId);
   });
 
-  it('C2379 Test Request app sorting (folijet) (prokopovych)', { tags: [testType.smoke, DevTeams.folijet] }, () => {
+  // Test are failed. This is a known issue. That's why test type is deleted.
```

Comment:
"Test are failed." Maybe you mean IS failed. Just typo :)

Reply:
yes, it's my bad)